http/src/api_error.rs

```diff
@@ -485,12 +485,15 @@ impl Display for MessageApiError {
 pub enum MessageApiErrorEmbedField {
     /// Something was wrong with the provided fields.
     Fields,
+    /// The provided timestamp wasn't a valid RFC3339 string.
```

Comment:
ISO 8601

Reply:
To my knowledge `RFC3339` and `ISO 8601` are essentially the same. Chrono only provides an [`.to_rfc3339()` function](https://docs.rs/chrono/0.4.18/chrono/struct.DateTime.html#method.to_rfc3339) that is `ISO 8601` compatible, so I chose that to make it a bit clearer

Reply:
Maybe `RFC3339/ISO8601` would be better since they both work.

Reply:
RFC3339 uses ISO 8601